src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs

```diff
@@ -933,8 +933,10 @@ public bool TryFormat(Span<char> destination, out int charsWritten, [StringSynta
         // IParsable
         //
 
+        /// <inheritdoc cref="IParsable{TSelf}.Parse(string, IFormatProvider?)" />
         public static TimeOnly Parse(string s, IFormatProvider? provider) => Parse(s, provider, DateTimeStyles.None);
 
+        /// <inheritdoc cref="IParsable{TSelf}.TryParse(string?, IFormatProvider?, out TSelf)" />
```

Comment:
This inheritdoc format is new to me, and the porting tool won't be able to detect it, so I will have to fix it. What is the purpose of cref here?

Reply:
I don't know, I just copied and pasted from similar places. I think you can just ignore it as if it is not there if you pick the right docs from `IParsable`.

Reply:
@carlossanlop, the `cref` indicates that the docs come from the specified member, it allows `inheritdoc` to be used when there is no implicit mapping.

Things like summary, returns, and remarks flow automatically. Things like `param` and typeparam` only flow where the names match between the two.